src/app/courses/courses-list-item/highlight.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, ElementRef, Input } from '@angular/core';
+import { CoursesListItem } from './courses-list-item-model';
+
+@Directive({
+  selector: '[vpHighlight]'
+})
+export class HighlightDirective {
+
+  @Input('vpHighlight') item: CoursesListItem;
+
+  constructor(private  element: ElementRef) { 
+  }
+
+  ngOnInit() {
+    const currentDate: Date = new Date();
+    const freshDate: Date = new Date();
+    freshDate.setDate(freshDate.getDate() - 14);
```

Comment:
make a const HOW_LONG_ITS_FRESH = 14, don't like magic numbers without any context, when you converting minutes and hours it's ok, here I don't think so